app/src/main/java/com/example/molliestephenson/tictactoe/MobilePlayer.java

```diff
@@ -0,0 +1,29 @@
+package com.example.molliestephenson.tictactoe;
+
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.Marks;
+
+public class MobilePlayer implements Player {
+
+    private Marks mark;
+
+    public MobilePlayer(Marks mark) {
+        this.mark = mark;
+    }
+
+    @Override
+    public Marks getMark() {
+        return mark;
+    }
+
+    @Override
+    public int getLocation(Board board) throws Exception {
+        return 0;
```

Comment:
Is this an initial stub implementation? It feels "weird" that the player just returns `0`? Is the value ignored?